### ISSUE TITLE
Popover: Axe accessibility issues:

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2371]
In Skills home page, After run on Axe accessibility tool scan found the observation for header menu related to PopoverTrigger component

 Magma Version - "react-magma-dom": "~4.12.0-next.5",

URL - https://dev.infosec-skills.com/portal/skills/home

Defect - https://infoseci.atlassian.net/browse/AAGL-1941

Suggestion from Tool - <div aria-haspopup="dialog" aria-expanded="false" aria-controls="778d9c4c-ada8-4a6e-a271-365b99f287cc_content" id="778d9c4c-ada8-4a6e-a271-365b99f287cc_trigger" tabindex="-1" class="css-6ayzi5-TriggerButtonContainer e3am2s00">
To solve this problem, you need to fix the following:
ARIA attribute is not allowed: aria-expanded="false"

Detail Info -
User Experience:
Assistive technologies such as screen readers rely on correctly defined ARIA roles and attributes to understand the purpose, state, and behaviour of interface elements. When an element is assigned an ARIA attribute that is not supported by its defined role, assistive technologies may misinterpret the element or ignore important information. This can lead to incorrect announcements, confusion for screen reader users, and difficulty in understanding or interacting with page components.
Repro Steps:
Open URL: https://dev.infosec-skills.com/portal/skills/home and login with valid credentials.
Navigate to the Header Section.
Run Accessibility Insights or Axe tool.
Review the accessibility issues reported by the tool.

Observe the issue stating “Ensure an element's role supports its ARIA attributes.”
Actual Result:
Some elements contain ARIA attributes that are not supported by their assigned roles. This causes assistive technologies to misinterpret the element semantics or ignore the ARIA attributes, resulting in inconsistent or incorrect announcements.
Expected Result:
All ARIA attributes should be used only with roles that support those attributes according to ARIA specifications. Elements should follow proper role-attribute relationships so assistive technologies can correctly interpret and announce them.
Attachment:
Please refer to the attachment section.
WCAG Guideline:
4.1.2 Name, Role, Value [A] [2.0]
Test Environment:
Environment: Windows 11 | Chrome v145.0.7632.46 | Accessibility Insights Tool v2.46.0 | Axe tool v4.11.

<img width="3024" height="1964" alt="Image" src="https://github.com/user-attachments/assets/9827b11d-c8e6-4e0d-b1ec-28b3cf562c90" />

https://cengage.slack.com/archives/C8Y4FR63U/p1774358931935269

**NOTE: Contact the team when the issue is resolved.**

**Comment by olehnoskov-cengage** [comment]
**Tested on:**
**- V4:**
Storybook: https://storybook-preview-v4-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/popover--profile-example

<img width="1843" height="1013" alt="Image" src="https://github.com/user-attachments/assets/88bea00e-e6ca-4004-970d-73c3ebc2757a" />

**- V5:** 
Storybook: https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/popover--profile-example
<img width="1913" height="1037" alt="Image" src="https://github.com/user-attachments/assets/6757b704-b695-48c9-a7f7-c1462d6cc53d" />
Everything works as expected. No bugs found.
